contracts/poap/src/contract.rs

```diff
@@ -0,0 +1,443 @@
+use crate::error::ContractError;
```

Comment:
We should have unit tests for contracts. `contract_tests` should be for checking the state outside of this contract.